src/core/MouseCapture.tsx

```diff
@@ -102,53 +131,52 @@ export default class MouseCapture extends React.Component<Props, State> {
         const { xScale, yScale } = this._createPhysicalToLogicalScales();
         this.props.onDragStart(xScale(e.clientX), yScale(e.clientY), e);
       }
+
+      this._removeWindowMouseEventHandlers();
+      this._addWindowMouseEventHandlers();
     }
   };
 
-  private _maybeDispatchDragHandler(e: React.MouseEvent, handler: (xPct: number, yPct: number, e: React.MouseEvent) => void) {
-    if (e.button === LEFT_MOUSE_BUTTON && handler && this.state.mouseDownClientX != null) {
+  private _onMouseMoveInCaptureArea = (e: React.MouseEvent) => {
+    if (this.props.onHover) {
       const { xScale, yScale } = this._createPhysicalToLogicalScales();
-      handler(
-        xScale(e.clientX),
-        yScale(e.clientY),
-        e
-      );
-    }
-  }
+      this.props.onHover(xScale(e.clientX), yScale(e.clientY), e);
 
-  private _onMouseMove = (e: React.MouseEvent) => {
-    this._maybeDispatchDragHandler(e, this.props.onDrag);
+      // If onHover exists, event.stopPropagation is called and the window event does not get called.
```

Comment:
I have not been able to notice any behavioral bugs with the InteractionCaptureLayer without the `stopPropagation` calls, so removed them.